scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/SoftKeywords.scala

```diff
@@ -69,7 +69,7 @@ class SoftKeywords(dialect: Dialect) {
   @classifier
   trait KwEnd {
     val name = "end"
-    @inline def isEnabled = dialect.allowSignificantIndentation
+    @inline def isEnabled = dialect.allowEndMarker
```

Comment:
Actually, do we need a separate dialect option for this? I don't think it will be removed in the future.

Reply:
that i don't know 🙂 should we remove it instead?

Reply:
to be clear: i haven't added the flag, I'm simply making its use consistent.

Reply:
Maybe let's leave it, just in case Martin decides against end markers in the future :sweat_smile: